### ISSUE TITLE
mkfs.ext4 can ask questions

### ISSUE THREAD

**Issue by ial0** [Antergos/Cnchi#416]
I recently got an issue, while trying to format a partition, mkfs complained about finding an msdos table on the partition and was asking whether to continue, cnchi doesn't  not handle this situation.  Adding force (-F) to mkfs might be a possible solution.

**Comment by karasu** [comment]
```
-F     Force mke2fs to create  a  filesystem,  even  if  the  specified
              device is not a partition on a block special device, or if other
              parameters do not make sense.  In order to force mke2fs to  cre‐
              ate  a filesystem even if the filesystem appears to be in use or
              is mounted (a truly dangerous thing to do), this option must  be
              specified twice.
```

Thanks!

**Comment by karasu** [comment]
Fixed in 0.13.24
